x/evm/core/vm/extended_eips.go

```diff
@@ -0,0 +1,25 @@
+package vm
```

Comment:
let's move this to the `app/` folder and add the license

Reply:
Yes will do, this was just a POC to check that everything was working fine and test manually the gas was updated properly!